velox/functions/prestosql/aggregates/ArbitraryAggregate.cpp

```diff
@@ -16,8 +16,8 @@
 
 #include "velox/exec/Aggregate.h"
 #include "velox/expression/FunctionSignature.h"
+#include "velox/functions/lib/SimpleNumericAggregate.h"
 #include "velox/functions/prestosql/aggregates/AggregateNames.h"
-#include "velox/functions/prestosql/aggregates/SimpleNumericAggregate.h"
 #include "velox/functions/prestosql/aggregates/SingleValueAccumulator.h"
```

Comment:
Should we move SingleValueAccumulator as well? We don't want to have "lib" depend on "prestosql".